src/maxminddb.c

```diff
@@ -795,6 +797,8 @@ static int populate_description_metadata(MMDB_s *mmdb,
             " status = %d (%s)",
             status,
             MMDB_strerror(status));
+        if (status != MMDB_OUT_OF_MEMORY_ERROR)
+            MMDB_free_entry_data_list(member);
```

Comment:
This requires it as it is; we can't use `goto cleanup;` directly, but maybe like this.

```c
        if (status != MMDB_OUT_OF_MEMORY_ERROR)
            goto cleanup;
```